dist/pom.xml

```diff
@@ -15,6 +15,16 @@
 
   <name>Zeebe Distribution</name>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>co.elastic.clients</groupId>
+        <artifactId>elasticsearch-java</artifactId>
+        <version>8.14.2</version>
```

Comment:
❓ From what I can see from our dependencies https://github.com/camunda/camunda/blob/a2cd92fe25a728b72b4143e93022cd77a8146ac3/parent/pom.xml#L56 is that this is a minor update, can we be sure that this works with different server version, etc.?